samples/http-client-examples/server.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const app = express();
+const port = 3000;
+
+app.head("/csrf-token", async (req, res) => {
```

Comment:
[pp]I would assume there are also `@types/express` so you could make this TS and get some type safety.